doctr/models/preprocessor/pytorch.py

```diff
@@ -35,7 +35,6 @@ def __init__(
         batch_size: int,
         mean: Tuple[float, float, float] = (0.5, 0.5, 0.5),
         std: Tuple[float, float, float] = (1.0, 1.0, 1.0),
-        fp16: bool = False,
```

Comment:
No more used or was it here only for signature consistency for something else ?

Reply:
it's unused from an old try now it's simply `.cuda().half()`  to the predictor or the TF equivalent :)

Reply:
ok, thanks